manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -494,10 +493,10 @@ func (r *FybrikApplicationReconciler) constructDataInfo(req *DataInfo, appContex
 	configEvaluatorInput.Workload.Cluster = workloadCluster
 	configEvaluatorInput.Request = CreateDataRequest(input, req.Context, &req.DataDetails.ResourceMetadata)
 
-	// Read policies for data that is processed in the workload geography
+	// Policies for data that is processed in the workload geography
 	if configEvaluatorInput.Request.Usage == taxonomy.ReadFlow {
```

Comment:
Is this still need to be `taxonomy.ReadFlow`?

Reply:
I'm rewriting this function. However, there are two main issues here:
- no 1:1 matching from flow to action type, i.e. there is no such thing as checking if copy is allowed (only WRITE action is supported by the connectors, no COPY).
- For Copy and Write flows we can't send a request to the policy manager because we don't know the final destination, it is checked later when a storage account is determined.
At this stage we can check only Read and Delete permissions that do not depend on the data path construction.